<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IoTTestbed</class>
 <widget class="QMainWindow" name="IoTTestbed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IoT Testbed</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="leftColumnLayout">
      <item>
       <layout class="QVBoxLayout" name="choicesContainer">
        <item>
         <widget class="QLabel" name="categoriesLabel">
          <property name="text">
           <string>Category</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="categoriesCombo"/>
        </item>
        <item>
         <widget class="QLabel" name="productsLabel">
          <property name="text">
           <string>Product</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="productsCombo"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="empty">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="rightColumnLayout">
      <item>
       <widget class="QLabel" name="resultsLabel">
        <property name="text">
         <string>Test Results</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="resultsTextArea"/>
      </item>
      <item>
       <widget class="QPushButton" name="startTestingButton">
        <property name="text">
         <string>Start Testing</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>728</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="newTestButton"/>
    <addaction name="separator"/>
    <addaction name="exitButton"/>
   </widget>
   <addaction name="menuAdd"/>
  </widget>
  <action name="newTestButton">
   <property name="text">
    <string>New Test</string>
   </property>
  </action>
  <action name="exitButton">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
